doc/getting-started.md

```diff
@@ -97,54 +126,79 @@ curl -iX POST \
 '
 ```
 
+The device `motion001` has been provisioned to persist data to the Context Broker as an entity of `type=Motion` (instead
+of the default `type=Thing`). The destination entity is identified by the `entity_name` field, which is set to
+`urn:ngsi-ld:Motion:001`. The device has a single attribute mapping for a measurement `c` to be renamed to `count`,
+additionally to one defined in the group mapping (`temperature`). The device also has a static attribute `refStore`
+which is a relationship to the entity `urn:ngsi-ld:Store:001`.
+
 This information is combined with the common config group information whenever a request is received at the South port
 of the IoT Agent and used to create or update the relevant entity in the Context Broker.
 
-#### Receiving a measure from a known Device
+## Receiving measures from devices
 
-For example, imagine we are using the Ultralight IoT Agent and the following request is sent to the South port:
+In order to see the complete process from provisioning the groups and devices to receiving measures, we will show how a
+device can send a measure to the IoT Agent. In this case, we will use the IoTA JSON as an example, using the HTTP
+transport protocol. To reproduce the measure sending, you can use the following `curl` commands.
+
+The device measures are sent to the South port of the IoT Agent which is listening in the port `7896`.
+
+### Receiving a measure from a known Device
+
+In this case, the device has been provisioned previously. We will use the `motion001` device defined in the previous
+example. To simulate the device sending a measure, the following request is sent to the South port:
 
 ```bash
 curl -iX POST \
-  'http://localhost:7896/iot/d?k=4jggokgpepnvsb2uv4s40d59ov&i=motion001' \
-  -H 'Content-Type: text/plain' \
-  -d 'c|1'
+  'http://localhost:7896/iot/json?k=4jggokgpepnvsb2uv4s40d59ov&i=motion001' \
+  -H 'Content-Type: application/json' \
+  -d '{"t":23,"c":1}'
 ```
 
 The IoT Agent South port is listening to the path defined in the config group, and the API key is recognized to match.
 Because the `device_id` is also recognized, the provisioned device configuration is used and its settings are combined
-with the config group. A mapping has been found to rename the `c` measurement to `count` - the following context entity
-is created in the context broker:
+with the config group.
+
+Mapping has been found to rename the `c` measurement to `count` and the `t` measurement to `temperature`. The following
+context entity is created in the context broker:
```

Comment:
Try to avoid the *rename* term here, as suggested in other comments above.

Reply:
Fixed here: https://github.com/telefonicaid/iotagent-node-lib/pull/1646/commits/3c6e239e367713c2a77d9f926b47261ef479c712